p/scripts/global_view.js

```diff
@@ -19,7 +19,7 @@ function load_panel(link) {
 			return;
 		}
 		const html = this.response;
-		const foreign = html.querySelectorAll('.nav_menu, #stream .day, #stream .flux, #stream .pagination, #stream.prompt');
+		const foreign = html.querySelectorAll('.nav_menu, #stream .day, #stream .flux, #stream #mark-read-pagination, #stream.prompt');
```

Comment:
`#stream #mark-read-pagination` can probably be simplified to `#mark-read-pagination`.
I have not yet looked at this line in details